dashboard/app/assets/javascripts/header.js.erb

```diff
@@ -413,3 +419,35 @@ dashboard.header.updateTimestamp = function () {
     $('.project_updated_at').text("Not saved"); // TODO i18n
   }
 };
+
+function getSummarizedProgressForAnonymousUser () {
+  var summarizedProgress = {};
+  var levelProgress = {};
+
+  summarizedProgress.lines = $.cookie('lines');
+
+  var allLevelsProgress = JSON.parse(decodeURIComponent($.cookie('progress')));
+  for(var level in allLevelsProgress) {
+    var individualLevelProgressDescription = '';
```

Comment:
I'm of the belief that verbose names are generally better as they remove the need for comments describing what a variable is. I can still make the change to fix this though.

If we're worried about file size, then we absolutely should be minifying names. Let's put that in the backlog.